charts/reform-scan-blob-router/values.yaml

```diff
@@ -10,6 +10,9 @@ java:
         - error-notifications-password
         - error-notifications-url
         - error-notifications-username
+        - reports-email-username
+        - reports-email-password
+        - reports-recipients
```

Comment:
This one doesn't exist in bootstrap.yaml

Reply:
bootstrap is not necessary when doing chart updates